server/src/modules/data/usecases/getTransformationStats/dependencies.ts

```diff
@@ -1,109 +1,94 @@
 
-const selectPlacesOuvertesScolaire =
-  (eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">) =>
-  (partitionBy?: AnyColumnWithTable<DB, "dataEtablissement">) =>
-    eb.fn
-      .sum<number>(
-        sql`GREATEST(${eb.ref("demande.capaciteScolaire")}
+const selectPlacesOuvertesScolaire = (
+  eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">
+) =>
+  eb.fn.sum<number>(
+    sql`GREATEST(${eb.ref("demande.capaciteScolaire")}
         - ${eb.ref("demande.capaciteScolaireActuelle")}, 0)`
-      )
-      .over((ob) => (partitionBy ? ob.partitionBy(partitionBy) : ob));
+  );
 
-const selectPlacesFermeesScolaire =
-  (eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">) =>
-  (partitionBy?: AnyColumnWithTable<DB, "dataEtablissement">) =>
-    eb.fn
-      .sum<number>(
-        sql`GREATEST(${eb.ref("demande.capaciteScolaireActuelle")}
+const selectPlacesFermeesScolaire = (
+  eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">
+) =>
+  eb.fn.sum<number>(
+    sql`GREATEST(${eb.ref("demande.capaciteScolaireActuelle")}
       - ${eb.ref("demande.capaciteScolaire")}, 0)`
-      )
-      .over((ob) => (partitionBy ? ob.partitionBy(partitionBy) : ob));
+  );
 
-const selectPlacesOuvertesApprentissage =
-  (eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">) =>
-  (partitionBy?: AnyColumnWithTable<DB, "dataEtablissement">) =>
-    eb.fn
-      .sum<number>(
-        sql`GREATEST(${eb.ref("demande.capaciteApprentissage")}
+const selectPlacesOuvertesApprentissage = (
+  eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">
+) =>
+  eb.fn.sum<number>(
+    sql`GREATEST(${eb.ref("demande.capaciteApprentissage")}
       - ${eb.ref("demande.capaciteApprentissageActuelle")}, 0)`
-      )
-      .over((ob) => (partitionBy ? ob.partitionBy(partitionBy) : ob));
+  );
 
-const selectPlacesFermeesApprentissage =
-  (eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">) =>
-  (partitionBy?: AnyColumnWithTable<DB, "dataEtablissement">) =>
-    eb.fn
-      .sum<number>(
-        sql`GREATEST(${eb.ref("demande.capaciteApprentissageActuelle")}
+const selectPlacesFermeesApprentissage = (
+  eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">
+) =>
+  eb.fn.sum<number>(
+    sql`GREATEST(${eb.ref("demande.capaciteApprentissageActuelle")}
     - ${eb.ref("demande.capaciteApprentissage")}, 0)`
-      )
-      .over((ob) => (partitionBy ? ob.partitionBy(partitionBy) : ob));
+  );
 
-const selectNbDemandes =
-  (eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">) =>
-  (partitionBy?: AnyColumnWithTable<DB, "dataEtablissement">) =>
-    eb.fn
-      .count<number>("demande.id")
-      .over((ob) => (partitionBy ? ob.partitionBy(partitionBy) : ob));
+const selectNbDemandes = (
+  eb: ExpressionBuilder<DB, "demande" | "dataEtablissement">
+) => eb.fn.count<number>("demande.id");
 
-const getTransformationStatsQuery = ({
+export const getDataScoped = async ({
```

Comment:
```suggestion
const getDataScoped = async ({
```

car exporté dans les dependencies en bas